### ISSUE TITLE
[Paper Note] Scaling Beyond Masked Diffusion Language Models, Subham Sekhar Sahoo+, arXiv'26, 2026.02

### ISSUE THREAD

**Issue by AkihikoWatanabe** [AkihikoWatanabe/paper_notes#4604]
# URL
- https://arxiv.org/abs/2602.15014
# Authors
  - Subham Sekhar Sahoo
  - Jean-Marie Lemercier
  - Zhihan Yang
  - Justin Deschenaux
  - Jingyu Liu
  - John Thickstun
  - Ante Jukic
# Abstract
  - Diffusion language models are a promising alternative to autoregressive models due to their potential for faster generation. Among discrete diffusion approaches, Masked diffusion currently dominates, largely driven by strong perplexity on language modeling benchmarks. In this work, we present the first scaling law study of uniform-state and interpolating discrete diffusion methods. We also show that Masked diffusion models can be made approximately 12% more FLOPs-efficient when trained with a simple cross-entropy objective. We find that perplexity is informative within a diffusion family but can be misleading across families, where models with worse likelihood scaling may be preferable due to faster and more practical sampling, as reflected by the speed-quality Pareto frontier. These results challenge the view that Masked diffusion is categorically the future of diffusion language modeling and that perplexity alone suffices for cross-algorithm comparison. Scaling all methods to 1.7B parameters, we show that uniform-state diffusion remains competitive on likelihood-based benchmarks and outperforms autoregressive and Masked diffusion models on GSM8K, despite worse validation perplexity. We provide the code, model checkpoints, and video tutorials on the project page: http://s-sahoo.github.io/scaling-dllms
# Translation (by gpt-5-nano)
- 拡散型言語モデルは、生成をより速く行える可能性があることから、自己回帰型モデルの有望な代替手段となっている。離散拡散アプローチの中では、マスクド拡散が現在優位に立っており、言語モデリングのベンチマークにおける高いパープレキシティの影響で主に推進されている。本研究では、一様状態拡散法と補間的離散拡散法の初のスケーリング法則に関する研究を提示する。また、単純なクロスエントロピー損失で訓練した場合、マスクド拡散モデルは約12％のFLOPs効率を高めることができる。パープレキシティは拡散ファミリー内では有用だが、ファミリー間では誤解を招くことがあり、尤度スケーリングが劣るモデルが、より速く実用的なサンプリングを可能にするため、速度と品質のパレート前線として反映されることがある。これらの結果は、マスクド拡散が拡散言語モデリングの未来を断定的に占めるという見解や、アルゴリズム間比較にはパープレキシティだけで十分であるという見解に挑戦する。すべての手法を17億パラメータへスケールさせると、一様状態拡散は尤度ベースのベンチマークで依然として競争力を保ち、GSM8K では自己回帰およびマスクド拡散モデルを上回る一方、検証データのパープレキシティは悪化する。コード、モデルチェックポイント、およびビデオチュートリアルはプロジェクトページに掲載しています: http://s-sahoo.github.io/scaling-dllms
# Summary (by gpt-4o-mini)
- 拡散型言語モデルは生成速度向上の可能性から自己回帰型モデルの代替手段となり、マスクド拡散が優位なアプローチとして注目されている。本研究では、一様状態拡散法と補間的離散拡散法のスケーリング法則を初めて提示し、マスクド拡散モデルが約12％のFLOPs効率向上を示すことを報告。パープレキシティは拡散ファミリー内で有用だが、他のファミリーとの比較では誤解を招くことがある。全手法を17億パラメータにスケールすると、一様状態拡散は依然として競争力を保ちつつ、GSM8Kで他モデルを上回りつつパープレキシティは悪化する結果となった。

**Comment by AkihikoWatanabe** [comment]
元ポスト: https://x.com/zhihanyang_/status/2023624944928633280?s=20

**Comment by AkihikoWatanabe** [comment]
pj page:https://s-sahoo.com/scaling-dllms/

**Comment by AkihikoWatanabe** [comment]
Masked Diffusion Language Model (MDLM)はperplexityの観点では高い性能が出るが、異なるDiffusion Algorithmを比較する上でPerplexityが良い指標なのか？がResearch Questionで、3種類の拡散モデル[^1]に基づくモデルを同一の計算量の元でスケーリング時の挙動を分析したとのこと。

その結果、計算量を投入すればするほどARモデルのような綺麗なスケーリング則が全てのモデルで見出されたが、PerplexityがARと同等の性能に到達するためには、MDLMが14--16倍、Duoが23倍、Eso-LMが32倍の計算量を要した。
Perplexityの観点ではMDLMが良さそうだが、Perplexityが良いからといって、サンプル効率、あるいは下流タスクの性能が良いとは限らないため追加の分析を実施。

スループット（token/sec）を変化させて検証したところ、ARは品質が高いが遅く、スループットが高い領域ではDuoがサンプル効率と品質のパレート最適であることがわかり、中くらいの領域ではEso-LMがパレート最適、低い領域でさARがパレート最適であり、スループットと品質の観点ではMDLMは劣ることがわかった。

その後、パラメータ数を1.7Bに固定し、Nemotron Pretrainingデータセットで事前学習をし、zeroshotでの（尤度ベースでの）下流タスクの性能を見ると、MDLMよりもDuoの方が5/7のベンチマークで性能が良く、その後GSM8KでSFTすると、DuoのPerplexityは低かったにも関わらず、全てのモデルを上回った。

[^1]: MDLMに加えて、Uniform-state Diffusion (Duo), Interpolating Diffusion(Eso-LM)というモデルで比較しているようである。この辺はあまり詳しくないので勉強したい。

という話が元ポストに書かれている。
